pkg/volume/util/atomic_writer.go

```diff
@@ -86,25 +86,35 @@ const (
 
 // Write does an atomic projection of the given payload into the writer's target
 // directory.  Input paths must not begin with '..'.
+// setPerms is an optional pointer to a function that caller can provide to set the
+// permissions of the newly created files before they are published. The function is
+// passed tsDir which is full path to timestamp subdir.
```

Comment:
full path or relative to the writer's root?

Reply:
Absolute path is now replaced with the timestamp directory name.